source/guide_passbolt.rst

```diff
@@ -0,0 +1,128 @@
+.. highlight:: console
+
+.. author:: Thomas Kammerer <https://kammerer.tk/>
+
+.. tag:: lang-php
+.. tag:: web
+
+.. sidebar:: Logo
+
+  .. image:: _static/images/passbolt.png
+      :align: center
+
+#########
+Passbolt
+#########
+
+.. tag_list::
+
+Passbolt_ 
+
+The password manager your team was waiting for. Free, open source, self-hosted, extensible, OpenPGP based.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * :manual:`PHP <lang-php>`
+  * :manual:`MySQL <database-mysql>`
+  * :manual:`domains <web-domains>`
+
+License
+=======
+
+Passbolt is released under the `AGPL-3.0 license`_.
+
+Prerequisites
+=============
+
+We’re using :manual:`PHP <lang-php>` in the stable version 7.2. Since new Uberspaces are currently setup with PHP 7.1 by default you need to set this version manually:
+
+::
+
+ [isabell@stardust ~]$ uberspace tools version use php 7.2
+ Selected PHP version 7.2
+ The new configuration is adapted immediately. Patch updates will be applied automatically.
+ [isabell@stardust ~]$
+
+.. include:: includes/my-print-defaults.rst
+
+Your website domain needs to be set up:
+
+.. include:: includes/web-domain-list.rst
+
+Create the database:
+
+ [isabell@stardust ~]$ echo "CREATE DATABASE isabell_passbolt CHARACTER SET utf8mb4 COLLATE utf8mb4_unicode_ci;" | mysql
+ [isabell@stardust ~]$
+
+Installation
+============
+
+To install Passbolt we clone the current version using Git. ``cd`` to your :manual:`DocumentRoot <web-documentroot>` so the cloned folder will under your ``html``.
+
+.. code-block:: console
+ :emphasize-lines: 2
+
+ [isabell@stardust ~]$ cd /var/www/virtual/$USER/html/
+ [isabell@stardust html]$ git clone https://github.com/passbolt/passbolt_api.git .
+ Cloning into '.'...
+ […]
+ [isabell@stardust ~]$
+
+Configuration
+=============
+
+Generate an OpenPGP key:
+
+.. warning:: Do not set a passphrase or an expiration date.
+
+Save your fingerprint and replace ``SERVER_KEY@EMAIL.TEST`` with your email.
+.. code-block:: console
+ :emphasize-lines: 2,3,4
+
+ [isabell@stardust ~]$ gpg --gen-key
+ [isabell@stardust ~]$ gpg --list-keys --fingerprint
+ [isabell@stardust ~]$ gpg --armor --export-secret-keys SERVER_KEY@EMAIL.TEST > /var/www/virtual/$USER/html/config/gpg/serverkey_private.asc
```

Comment:
This exposes the private key to the public internet. It seems like only the `webroot` directory should end up in `~/html`. Can you check this and adapt the code?